CHANGELOG.rst

```diff
@@ -2,6 +2,17 @@
 Wijzigingen
 ===========
 
+1.3.0-rc5 (2022-10-13)
```

Comment:
Hier staat versie 1.3.0-rc5 - in bumpversion staat echter 1.2.0-rc5.

Dat moet hetzelfde zijn.